cadence/contracts/ViewResolver.cdc

```diff
@@ -0,0 +1,59 @@
+// Taken from the NFT Metadata standard, this contract exposes an interface to let 
+// anyone borrow a contract and resolve views on it.
+//
+// This will allow you to obtain information about a contract without necessarily knowing anything about it.
+// All you need is its address and name and you're good to go!
+access(all) contract interface ViewResolver {
```

Comment:
Is the Cadence 1.0 version of this not and other core contracts not yet deployed to the emulator and testnet? I see an address listed for testnet below

Reply:
Not yet on testnet, but the migrated addresses should be the same AFAIK.

Reply:
I'll spin up a feature branch from main and target this PR against that so merging doesn't break anything